### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by NikuPAN**
Update README.md supported Unity versions, also updated the installation tutorial with a better, easier understandable steps followed by some screenshots. Furthermore, corrected a lot of spelling mistakes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,33 +6,52 @@ This project demonstrates the use of Euclideon udSDK developer Kit (udSDK SDK) w
...
+```
+- 2019.3.4f1
+- 2020.1.9f1
+- 2020.3.18f1
+- 2021.1.22f1
+```
+It may work in other versions of Unity, but no guarantee is provided for this. Please ensure you have one of the tested versions installed.
 
 ### Installation - New or Existing Project
 The fastest way to install udSDK for Unity is to go [here](https://Euclideon.com/unity) and follow the onscreen instructions.
```
Comment (EthanVellacott) [inline README.md:29]: We should probably get rid of this link while we are changing stuff - it's old, not very helpful (the "instructions" it includes point you back to the Github)

File: README.md [hunk 2]
```diff
@@ -44,18 +63,25 @@ Photogrammetry model of the Gold Coast courtesy of [Aerometrex](https://aerometr
...
 - Driving demo -Showing the usage of the udSDK collider object to make local mesh colliders for physics simulation
-- Filter Demo -  demonstrating the use of a query filter to selectively render volumes of the point cloud
+![image](https://user-images.githubusercontent.com/28862516/134598026-1de917c0-17c5-4938-b71e-ada5b4a72f43.png)
+- Filter Demo - demonstrating the use of a query filter to selectively render volumes of the point cloud
+![image](https://user-images.githubusercontent.com/28862516/134598173-339c056e-f665-48ee-83d4-e9fa703c5bd8.png)
 - Raycasting Example - Showing usage of the udSDK Collider to estimate surfaces in front of the camera for raycasting.
+![image](https://user-images.githubusercontent.com/28862516/134598278-33bb0947-b4e6-4a1f-ad0c-170b29e16808.png)
 - Projects demo - Example of importing udStream Projects including a uds and POIs into Unity
+![image](https://user-images.githubusercontent.com/28862516/134598379-9ef3b97d-041f-45df-afef-8c6a6be6c45c.png)
+- DepthOfField - Example of visual effect of using the depth buffer to simlulate camera lens effect
```
Comment (EthanVellacott) [inline README.md:77]: *simulate

Comment (NikuPAN) [reply README.md]: Solved

File: README.md [hunk 3]
```diff
@@ -6,33 +6,50 @@ This project demonstrates the use of Euclideon udSDK developer Kit (udSDK SDK) w
...
 ## Quickstart guide 
 
 **The Unity-udSDK sample requires a free Euclideon Account to use, licenses can be obtained [here](https://www.euclideon.com/udsdk/)** 
-The udSDK is tested with Unity 2019.3.4f1 - it may work in other versions of Unity, but we can't guarantee that it does. Please sneure you have Unity 2019.3.4f1 installed.
-
-### Installation - New or Existing Project
-The fastest way to install udSDK for Unity is to go [here](https://Euclideon.com/unity) and follow the onscreen instructions.
+The udSDK is tested with Unity on following versions:
+```
+- 2019.3.4f1
```
Comment (int-Frank) [inline README.md:21]: Start with the most current first

**Review by EthanVellacott — APPROVED** [review]

**Review by int-Frank — CHANGES_REQUESTED** [review]
Good :)

File: README.md (hunk 1 above)
Comment (NikuPAN) [reply README.md]: Done

**Pull request merged**